release-1.6/release_team.md

```diff
@@ -0,0 +1,12 @@
+|  **Role** | **Name** |
```

Comment:
Good idea. They are also may be useful at the [spreadsheet](https://docs.google.com/spreadsheets/d/1OvBDDzVeqzktyGZSD4-eMfnJgJros-86rgHQrrYjP-s/edit#gid=0).